lib/fenrir_view/components/fenrir_view_navbar/fenrir_view_navbar.js

```diff
@@ -1,119 +1,47 @@
+// @ts-check
 // ----------------------------------------\\
-// modules: navbar
+// System component: navbar
 // ----------------------------------------\\
-(function($) {
-  if (!$.fenrirView) {
-    $.fenrirView = {};
-  }
 
-  $.fenrirView.hideNavbarDropdown = function() {
-    $('.js-fenrir-view-navbar__item').removeClass('is-open');
-  }
+if (!designSystem) {
+  // @ts-ignore eslint-disable-next-line vars-on-top
+  var designSystem = {};
```

Comment:
How do you feel about making this more like we would do in Ruby on Charlie? `var designSystem = designSystem || {};`

Reply:
Hmm, good idea, not sure why we haven't done that before 🤔

I'll leave this as is for now, as I'm not sure it matter when these are moved to webpack.